rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -425,7 +432,7 @@ def persist(self, file_name: Text, model_dir: Text) -> Optional[Dict[Text, Any]]
                     utils.json_pickle(
                         featurizer_file,
                         {
-                            attribute: self.vectorizer[attribute].vocabulary_
+                            attribute: self._get_attribute_vocabulary(attribute)
                             if self._check_attribute_vocabulary(attribute)
```

Comment:
let's perform this check inside `_get_attribute_vocabulary`

Reply:
oh, you already do it, so this check is not needed

Reply:
It's happening inside, but we need to convert empty dict(in case of no vocabulary) to None anyways and hence the check.

Reply:
Fixed